build_scripts/packaging.conf

```diff
@@ -1,6 +1,6 @@
 # This config file is Bash shell syntax, and is included in all packaging scripts.
 
 # List of all Firebase products to include in the binary SDK package.
-readonly -a product_list=(admob analytics app auth database
+readonly -a product_list=(admob analytics app app_check auth database
```

Comment:
We don't want to update this one yet, since this is used for the release build.

Reply:
Reverted this file in later commit